ui/panels-plugin/src/plugins/stat-chart/StatChartPanel.tsx

```diff
@@ -81,9 +74,10 @@ const useChartData = (data: TimeSeriesData | undefined, calculation: Calculation
     return {
       calculatedValue,
       seriesData,
-      name,
+      // TODO: How is this name used in StatChart? Do we really need it? Should it be a chart spec option?
```

Comment:
This can be removed completely now, way [back in v0.4](https://github.com/perses/perses/blob/v0.4.2/ui/components/src/StatChart.tsx#L107) and before, I had an optional prop to show the name down beside the value, but it isn't used anymore since it's redundant with the panel header